modules/apple_support/0.12.1/MODULE.bazel

```diff
@@ -0,0 +1,9 @@
+module(
+    name = "apple_support",
+    version = "0.12.1",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.1.1")
+bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "stardoc", version = "0.5.0", repo_name = "io_bazel_stardoc", dev_dependency = True)
```

Comment:
Note that, there is a known issue with stardoc and Bzlmod: https://github.com/bazelbuild/bazel/issues/14140, but it's fine for now as a dev dependency.

Reply:
Ah yea, I also filed https://github.com/bazelbuild/stardoc/issues/117